docker/compose/docker-compose-nfs-mount.yml

```diff
@@ -53,6 +56,7 @@ services:
     environment:
       WAIT_FOR: zookeeper:2181
       ZK_URL: zookeeper:2181
+      REST_SERVER_PORT: 10080
```

Comment:
10080 seems to be a registered port: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

Reply:
I guess that makes Pravega incompatible with the Touhou fight games... :)

Reply:
10080 seems to have IANA status "Unofficial" (according to the last column in the link). After your feedback, I thought I should change it to 9091 - the default in Standalone mode, but then found two problems:
* 9091 (as well as 9090) are also showing as reserved there with unofficial status.
* I reckon there it might affect other areas that are outside the scope of this PR. 

I think we should take care of it separately outside this PR. What do you say?

Reply:
@ravisharda I agree. We are already using 10080 on other platforms (Kubernetes, Swarm). I don't know the reason behind mapping the default 9091 port to 10080, but I think we should discuss that in a separate thread.

Reply:
One reason I brought this up is that port scanner often complain when they see something in a registered port and in particular when it is something obscure and unexpected in a given environment. If we are already using it elsewhere, then I'm fine with keeping it as is.